webui/react/src/pages/ExperimentDetails.tsx

```diff
@@ -47,15 +47,19 @@ const ExperimentDetailsComp: React.FC = () => {
   const [ showCheckpoint, setShowCheckpoint ] = useState(false);
   const [ forkModalVisible, setForkModalVisible ] = useState(false);
   const [ forkModalConfig, setForkModalConfig ] = useState('Loading');
-  const [ experimentResponse, triggerExperimentRequest ] =
-    useRestApi<ExperimentDetailsParams, ExperimentDetails>(getExperimentDetails, { id });
   const storage = useStorage(STORAGE_PATH);
   const initLimit = storage.getWithDefault(STORAGE_LIMIT_KEY, MINIMUM_PAGE_SIZE);
   const initSorter: ApiSorter | null = storage.get(STORAGE_SORTER_KEY);
   const [ pageSize, setPageSize ] = useState(initLimit);
   const [ sorter, setSorter ] = useState<ApiSorter | null>(initSorter);
+  const [ restApiState, setRestApiState ] = useState<ApiState<ExperimentDetails>>({
```

Comment:
nit: name it according to the actual entity that we care about 'experimentDetails` instead of API state.